src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
```

Comment:
[CODE STYLE] - Avoid generic variable names like `styles`. Use more descriptive names that convey the purpose of the variable, such as `styleObject` or `cssProperties`.